packages/packages.csv

```diff
@@ -173,6 +171,7 @@ Microsoft.Extensions.TaskCache.Sources,noship,exclude
 Microsoft.Extensions.TypeNameHelper.Sources,noship,exclude
 Microsoft.Extensions.WebEncoders,ship,include
 Microsoft.Extensions.WebEncoders.Sources,noship,exclude
+Microsoft.Extensions.WebSockets.Internal,noship,exclude
```

Comment:
It needs to ship along with SignalR, so it needs to be 'ship'. It's `.Internal` to indicate that it is not currently supported on it's own. It's only "supported" via using SignalR.